deploy/helm/secret-operator/values.yaml

```diff
@@ -7,8 +7,8 @@ image:
 
 csiProvisioner:
   image:
-    repository: docker.stackable.tech/k8s/sig-storage/csi-provisioner
-    tag: v4.0.1
+    repository: docker.stackable.tech/k8s/csi-provisioner
+    tag: v5.0.1
```

Comment:
This is a new major version. Did you research if it's backwards compatible with the previous one?

Reply:
https://github.com/kubernetes-csi/external-provisioner/blob/v5.0.1/CHANGELOG/CHANGELOG-5.0.md

> The external-provisioner now needs permissions to patch persistentvolumes. Please update your RBACs appropriately.

I have no idea if we use/need that. But it does not seem like a straightforward upgrade at least.

Reply:
Looking at the PR it came from (https://github.com/kubernetes-csi/external-provisioner/pull/1155): yes we do need the permission, no it doesn't look like we have take the permission already.

Reply:
Yes, I'm updating it atm. I made another PR which was not the smartest thing. Put those together in this one now